routes/Login/createUserAdmin.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * Created by Kun on 7/14/2015.
+ */
+
```

Comment:
Can we just have one create user route that uses case statements for any unique logic? If we have two separate routes and need to make changes to shared logic, having the two routes requires that that change be made in both locations.